PayPal/src/main/java/com/braintreepayments/api/paypal/RecurringBillingAmountBreakdown.kt

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * A recurring billing amount breakdown.
+ *
+ * @property shipping - Cost of shipping the items
+ * @property taxTotal - Total tax amount applied to the transaction
+ * @property itemTotal - Total amount of the items before any taxes or discounts
+ */
+@Parcelize
+data class RecurringBillingAmountBreakdown(
+    val shipping: String,
```

Comment:
What do we think of renaming this to shipping total? Or replacing total with amount for all of them? May read better from the merchant integration side.

Reply:
I think that's a good call - I'll update the design for this and iOS